index.html

```diff
@@ -271,19 +271,18 @@ <h4>Skills</h4>
        <hr class="bottom-line">
        <div class="medias">
            <ul class="ul-socialmedia">
-               <li class="socialists"><a href="https://twitter.com/aidan-bn"><img src="./images/twitter.svg" alt="twitter"></a></li>
-               <li class="socialists"><a href="https://twitter.com/aidan-bn"><img src="./images/Linkedin.svg" alt="linkedin"></a></li>
-               <li class="socialists"><a href="https://maxcdn.com/aidan-bn"><img src="./images/medium.svg" alt="medium"></a></li>
-               <li class="socialists"><a href="https://github.com/aidan-bn"><img src="./images/github.svg" alt="github"></a></li>
-               <li class="socialists"><a href="https://angellist.com/aidan-bn"><img src="./images/angellist.svg" alt="angellist"></a></li>
+               <li class="socialists"><a href="https://twitter.com/bantezea"><img src="./images/twitter.svg" alt="twitter"></a></li>
+               <li class="socialists"><a href="https://www.linkedin.com/in/aidan-banteze-657973239/"><img src="./images/Linkedin.svg" alt="linkedin"></a></li>
+               <li class="socialists"><a href="https://github.com/Aidan-bn"><img src="./images/medium.svg" alt="medium"></a></li>
+               <li class="socialists"><a href="https://github.com/Aidan-bn"><img src="./images/github.svg" alt="github"></a></li>
+               <li class="socialists"><a href="https://www.linkedin.com/in/aidan-banteze-657973239/"><img src="./images/angellist.svg" alt="angellist"></a></li>
            </ul>
```

Comment:
- [ ] Good job so far :+1: But there are some issues you need to address . As the previous reviewer mentioned. Please fix the COLOR CONTRAST for the button to improve the web accessibility of your page to improve the user experience.

- [ ] The previous reviewer pointed out the missing form labels, But you did not implement them very well. Please fix them because labels are there to connect an explanatory text with a form element and this will make your web accessibility better :+1: 

- [ ] The previous reviewer mentioned  Langues / skills contrast. Please check your single project technologies list Contrast Errors because it is very low :+1: 

- [ ] [OPTIONAL]  I would like to recommend you include this extension in your browser's tab. You can follow [this link](https://stackoverflow.com/questions/4888377/how-to-add-a-browser-tab-icon-favicon-for-a-website). That will help you to check your website accessibility:smile: 

![Screenshot from 2022-07-13 09-59-11](https://user-images.githubusercontent.com/6615394/178707187-271fbef7-7a2b-428d-8ac2-93cc5eeddea5.png)